core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -51,6 +51,13 @@ trait DeltaSQLConfBase {
       .booleanConf
       .createOptional
 
+  val DELTA_COLLECT_STATS =
+    buildConf("stats.collect")
+      .internal()
```

Comment:
probably shouldn't be internal, eh?

Reply:
well...internal marks whether its value should show automatically when SQL command to list all conf shows up. I dont see this as a critical thing that we need to expose this.